### PR TITLE
fix: Fixes email body when sharing a chart by email

### LINKED ISSUE #14611: [share chart] share chart email message is not correct
When user share chart in dashboard, the email message is not correct.  See screenshot

### Expected results

share email show correct message with share chart information

### Actual results
share email message is not correct 

#### Screenshots

Screen Shot 2021-05-11 at 8 21 38 PM

#### How to reproduce the bug

repo steps
1, go to dashboard with charts
2, click on the 3 dot menu on one of the chart
3, select share chart by mail
4, wait for the mail to pop with message of the share

### Environment

(please complete the following information):

- superset version: master

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [Y] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [ ] I have reproduced the issue with at least the latest released version of superset.
- [ ] I have checked the issue tracker for the same issue and I haven't found one similar.

### PR DESCRIPTION
**Pull request by michael-s-molina**
### SUMMARY
Fixes https://github.com/apache/superset/issues/14611

@rusackas @junlincc 

### TEST PLAN
Check the original issue.

### ADDITIONAL INFORMATION
- [x] Has associated issue:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/dashboard/components/SliceHeaderControls/index.jsx [hunk 1]
```diff
@@ -266,6 +266,7 @@ class SliceHeaderControls extends React.PureComponent {
             copyMenuItemTitle={t('Copy chart URL')}
             emailMenuItemTitle={t('Share chart by email')}
             emailSubject={t('Superset chart')}
+            emailBody={t('Check out this chart: ')}
```
Comment (geido) [inline superset-frontend/src/dashboard/components/SliceHeaderControls/index.jsx:269]: Has anyone provided the copy? I am just wondering how do we go about copy in general when required. CC @rusackas

**Review by geido — APPROVED** [review]
LGTM!

**Comment by junlincc** [comment]
Screen Shot 2021-05-17 at 11 14 15 PM

work expected after the fix. thank you @michael-s-molina

**Review by junlincc — APPROVED** [review]

File: superset-frontend/src/dashboard/components/SliceHeaderControls/index.jsx (hunk 1 above)
Comment (rusackas) [reply superset-frontend/src/dashboard/components/SliceHeaderControls/index.jsx:269]: We don't have official copy-writing practices, per se. What we can do in instances like this one is consider (and potentially approve) the PR, and we could add the `need:design-review` label, which means that @mihir174 and/or @steejay will take a look from a _design_ perspective. If a body of copy gets large enough or seems contentious, we could take it to the dev@ mailing list for lazy consensus. 

This case seems small enough though that we could just merge it, and if anyone has better ideas, they can open a new PR to tweak it. This fixes the bug, in the meantime.

**Review by rusackas — APPROVED** [review]
LGTM!

**Pull request merged**